src/test/java/io/openshift/booster/OpenShiftIT.java

```diff
@@ -150,7 +154,10 @@ private void waitForApp() {
         await().atMost(5, TimeUnit.MINUTES)
                 .until(() -> {
                     try {
-                        final Response response = get(greetingServiceURI);
+                        final Response response =
```

Comment:
I guess this response variable isn't needed and the comparison could be similar to the one in testDConfigMapNotPresent just with a different code and exception handling

Reply:
Good catch, I have to admit that I didn't take a hard look at what the tests were actually doing. I'll update it :)